manager/src/main/java/org/openremote/manager/dashboard/DashboardStorageService.java

```diff
@@ -327,15 +314,11 @@ public boolean delete(String dashboardId, String realm, String userId) throws Il
         }
         return persistenceService.doReturningTransaction(em -> {
 
-            // Query the dashboards with the same ID (which is only 1), and that userId is able to EDIT
+            // Query the dashboards with the same ID (which is only 1)
```

Comment:
Same as my comments on update

Reply:
Seems inefficient fetching the whole dashboard, isn't the logic simple:
```
Dashboard Access != Private || (Private && user is owner)
```

Should super admin be able to view/delete anyone's dashboard (maybe a new issue needed to address this if not discussed already)?

Reply:
Don't think it's a good idea to authorize a `PUT` update based on their own Dashboard model input.. 😅 
Instead, I use query the dashboard along with their own permissions.

Unfortunately, similar to `AssetQuery`, these query checks tend to be quite extensive,
such as "Does the user have access to the assets in the dashboard?"
I think that is fine for now.